relayer/chains/mock/mock_chain_processor_test.go

```diff
@@ -102,6 +104,9 @@ func TestMockChainAndPathProcessors(t *testing.T) {
 func getMockMessages(channelKey processor.ChannelKey, mockSequence, mockSequenceCounterparty, lastSentMockMsgRecvCounterparty *uint64, lock *sync.Mutex) []mock.TransactionMessage {
 	lock.Lock()
 	defer lock.Unlock()
+	if *mockSequence-*mockSequenceCounterparty > 0 {
+		return []mock.TransactionMessage{}
+	}
```

Comment:
Couldn't this just be:
```
if mockSequence != mockSequenceCounterparty {
    return []mock.TransactionMessage{}
}
```

since afaik the sequence numbers should be the same?

Reply:
it can be 0 or -1, just want each side to only be able to increment one at a time.